### DISCUSSION TITLE
followers

### DISCUSSION THREAD

**Discussion by Fordowner49** [nroutasuo/level13#298] [Q&A] [answered]
How do i get a bigger max followers for my party

**Comment by nroutasuo** [comment, marked as answer]
For the party that ventures out together with you the limit is fixed. But the number of Followers that you can recruit can be increased by building and improving Inns.

**Reply by Fordowner49** [reply]
would that be by unlocking hospatality?

**Reply by nroutasuo** [reply]
Exactly!
